source/TimePicker.js

```diff
@@ -53,14 +53,37 @@
 		published: {
 			/**
 			* The meridiem text to display if [meridiemEnable]{@link moon.TimePicker#meridiemEnable}
-			* is `true`. The first item is used if the `hour` is less than `11`; otherwise, the
+			* is `true`. The first item is used if the `hour` is less than `12`; otherwise, the
 			* second is used.
 			*
 			* @type {String[]}
-			* @default ['AM','PM']
+			* @default [{name: 'AM', start: '00:00', end: '11:59'}, {name: 'PM', start: '12:00', end: '23:59'}]
 			* @public
 			*/
-			meridiems: ['AM','PM']
+			meridiems: [{name: 'AM', start: '00:00', end: '11:59'}, {name: 'PM', start: '12:00', end: '23:59'}]
+		},
+
+		/**
+		* Return index of meridiems array for given hour
+		*
+		* @param  {Number} hour - hour between 0 to 23
+		* @param  {Number} minute - minute between 0 to 59
+		* @return  {Number} index - index of this.meridiems
+		* @private
+		*/
+		getMeridiemIndex: function (hour, minute) {
+			var meridiems = this.meridiems;
+			for (var i = 0; i < meridiems.length; i++) {
+				var startArr = meridiems[i]['start'].substring(0,2) + meridiems[i]['start'].substring(3,5),
```

Comment:
Oh yes. I should change variable name.

And I realize that old code are remained. 
During some git processing, I made mistake